.github/workflows/build.yml

```diff
@@ -96,7 +96,7 @@ jobs:
       - # === Install Deps ===
         name: Install Deps
         shell: bash
-        timeout-minutes: 1
+        timeout-minutes: 10
```

Comment:
```suggestion
```

Unrelated to this PR. Also if this is taking more than a minute then we need to get the time down, not increase the timeout.

Reply:
It's a CI thing. Sometimes it's slow downloading and installing gozip, sometimes not. I wasn't experiencing any problems until yesterday, when it took me 5 tries before it worked on a 1 minute timeout.

Reply:
In my experience when it takes more than a minute it is going to timeout. Have you found that giving it more time sometimes works?